### DISCUSSION TITLE
[Feature] web, mobile: description over the asset

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#4794] [Feature Request]
### The feature

A nice Google Photos like feature would be to show description over the asset while slideshow (web) or when controls are hidden (mobile).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Vessel40** [comment]
+1 for this feature request. Not only the App-Which-Must-Not-Be-Named has this feature but also Photoprism has similar implementation.

**Comment by bilogic** [comment]
+1

**Comment by waclaw66** [comment]
Web part already implemented within https://github.com/immich-app/immich/pull/24627

**Comment by bo0tzz** [comment]
I don't see us adding the description overlay when everything else is hidden on mobile. Since the web side of the request has landed, I think we should close this thread.
